addons/mail/static/src/core/common/partner_compare.js

```diff
@@ -0,0 +1,94 @@
+/* @odoo-module */
+
+import { cleanTerm } from "@mail/utils/common/format";
+import { registry } from "@web/core/registry";
+
+/**
+ * Registry of functions to sort partner suggestions.
+ * The expected value is an object of the following form:
+ *    { sequence: number, fn: (partner1, partner2, { env: OdooEnv, searchTerm: string, thread?: Thread }) => number|undefined }
```

Comment:
now obsolete (function directly instead of object)

Reply:
updated